pkg/lifecycle/daemon/routes_v1.go

```diff
@@ -122,12 +123,19 @@ func (d *V1Routes) saveHelmValues(c *gin.Context) {
 		c.AbortWithError(http.StatusInternalServerError, errors.Wrap(err, "read file values.yaml"))
 	}
 
-	debug.Log("event", "serialize")
-	err = d.StateManager.SerializeHelmValues(request.Values, string(chartDefaultValues))
-	if err != nil {
-		debug.Log("event", "seralize.fail", "err", err)
+	debug.Log("event", "serialize.helmValues")
+	if err := d.StateManager.SerializeHelmValues(request.Values, string(chartDefaultValues)); err != nil {
+		debug.Log("event", "seralize.helmValues.fail", "err", err)
 		c.AbortWithError(http.StatusInternalServerError, errors.New("internal_server_error"))
 	}
+
+	debug.Log("event", "serialize.helmReleaseName")
+	if len(request.ReleaseName) > 0 {
+		if err := d.StateManager.SerializeReleaseName(request.ReleaseName); err != nil {
```

Comment:
why are we having the client set the release name?

Reply:
it's a client request 😅